snowflake/ConnectionWithSAStoken.txt

```diff
@@ -0,0 +1,28 @@
+-- creating database and warehouse --
```

Comment:
These two files have scripts that do almost the same, but they have significantly different names. Let's synchronize them.